modules/server/src/main/scala/navigate/server/tcs/TcsEpicsSystem.scala

```diff
@@ -45,6 +45,7 @@ object TcsEpicsSystem {
     val shuttersMoveCmd: Command2Channels[F, Double, Double]
     val ventGatesMoveCmd: Command2Channels[F, Double, Double]
     val sourceACmd: TargetCommandChannels[F]
+    val oiwfsCmd: TargetCommandChannels[F]
```

Comment:
The name will become ambiguous because we will add other OIWFS related commands in the future. I suggest to change it to `oiwfsTargetCmd`. Same for other names below.